app/upgrades/v4/upgrade.go

```diff
@@ -282,6 +268,7 @@ func ConvertOldSequencerToNew(old sequencertypes.Sequencer) sequencertypes.Seque
 		RollappId:    old.RollappId,
 		Status:       old.Status,
 		Tokens:       old.Tokens,
+		OptedIn:      true,
 		Metadata: sequencertypes.SequencerMetadata{
 			Moniker: old.Metadata.Moniker,
 			Details: old.Metadata.Details,
```

Comment:
i think you can remove todo about missing data 
```
TODO: regarding missing data - https://github.com/dymensionxyz/dymension/issues/987
```
(line 275, github doesn't allow to leave comments on unchanged lines)